py_gen/templates/init.py

```diff
@@ -30,7 +30,7 @@
 :: include('_autogen.py')
 
 import const
-:: for module in modules:
+:: for module in sorted(modules):
```

Comment:
could avoid the consumer-side sorting here by making this an `OrderedDefaultDict`?

https://github.com/floodlight/loxigen/blob/master/py_gen/codegen.py#L75

Reply:
thx; i didn't find OrderedDefaultDict, but was able to fake up an equivalent

Reply:
It's in `generic_utils`: https://github.com/floodlight/loxigen/blob/master/generic_utils.py#L151